city_scrapers/spiders/cle_cpc.py

```diff
@@ -0,0 +1,138 @@
+from datetime import datetime, time
+
+from city_scrapers_core.constants import COMMISSION, PASSED, TENTATIVE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class CleCpcSpider(CityScrapersSpider):
+    name = "cle_cpc"
+    agency = "Cleveland Community Police Commission"
+    timezone = "America/Chicago"
```

Comment:
Since we are scraping meetings in Cleveland, can you please set the correct timezone for this spider? This spider's timezone will be combined into the JSON data and pushed to the Azure container, and on our web app Documenters.org, we need this information to show the timezone next to the start time.

Reply:
Oooh, great catch, @lamle-ea. Yeah I totally overlooked that. Now fixed.

As an aside: I think I generated this spider through scrapy's `genspider` command, which of course uses our spider template from `city-scrapers-core`. When I start tweaking city-scrapers-core this is a good reminder to me that I probably should add some kind of option to tweak the timezone in the template spider.